api/src/main/java/org/example/users/models/User.java

```diff
@@ -60,8 +60,8 @@ public Sex getSex() {
         return sex;
     }
 
-    public void setSex(Sex sex) {
-        this.sex = sex;
+    public void setSex(String sex) {
+        this.sex.setName(sex);
```

Comment:
What is going on here. (this is going to give problems)

Reply:
that was a mistake, i've pushed a fix to this branch